pkg/flink/internal/controller/statement_controller_test.go

```diff
@@ -231,17 +227,73 @@ func (s *StatementControllerTestSuite) TestExecuteStatementPrintsNoWarningForSta
 	cupaloy.SnapshotT(s.T(), stdout)
 }
 
+func (s *StatementControllerTestSuite) TestExecuteStatementWaitsForCompletedState() {
+	statementToExecute := "select 1;"
```

Comment:
maybe it makes sense to replicate these tests for insert into and create table? 

We know that the behavior should be exactly the same, but just as a guardrail to make sure the test fails if it changes for different types of queries

Reply:
I see what you mean but it won't really help, since the logic is not influenced by the statements text, but only by what's inside the backend response. So the test would look identical

Reply:
I can change the query text to `insert into` though, that should be less confusing